lib/core/utils/normalize-node.js

```diff
@@ -0,0 +1,22 @@
+import AbstractVirtualNode from '../base/virtual-node/abstract-virtual-node';
+import getNodeFromTree from './get-node-from-tree';
+
+/**
+ * Get the virtual node and actual node from a parameter that could be either.
+ *
+ * @param {VirtualNode|Element} element
+ * @return {Object} The Virtual Node and actual node
+ */
+export default function normalizeNode(element) {
+  if (element instanceof AbstractVirtualNode) {
+    return {
+      vNode: element,
+      node: element.actualNode
```

Comment:
I would like us to get a little more consistent about what we're calling different nodes. I very much regret "actualNode" as a term. `domNode` would have been much better. How about this:

1. We barely use virtualNode anymore, I think that's okay so lets stick with vNode
2. When we talk about a DOM node, lets call it that; domNode, domParent, whatever.
3. When something can be either a domNode or a virtualNode we just use `node`. That's only for passing this in

```suggestion
      domNode: element.actualNode
```